### DISCUSSION TITLE
Delegate with a singleton instance

### DISCUSSION THREAD

**Discussion by thesrs02** [atomiks/tippyjs#1052] [Q&A]
Hi is there any documentation on how to use **createSingleton** with the **delegate** method? I tried to find documentation on this and have also tried to test some parts of it but I was not able to make it work.

**Comment by segevfiner** [comment]
@atomiks So I guess this isn't currently supported? Having a delegate that manages it's children under a singleton?

**Comment by sanchi-t** [comment]
@atomiks is there any plan to include this feature in the future?
